### DISCUSSION TITLE
umi4 public目录下的图片资源路径问题

### DISCUSSION THREAD

**Discussion by zhangfinn** [umijs/umi#8320] [General]
页面里好像不支持之前那种background-image: url(/xxxxxx.png)的写法。需要../../  有没有解决办法（之前页面里都是简写）

**Comment by fz6m** [comment]
最新版本已支持使用 `/*` 开头的绝对路径引用静态资源。

**Reply by ihaichao** [reply]
你好，在ts里通过import引入路径怎么写，尝试了绝对路径找不到
